Documentation/kubernetes.md

```diff
@@ -191,6 +191,8 @@ kind: Cluster
 metadata:
   name: rook
   namespace: rook
+  finalizers:
+  - cluster.rook.io
```

Comment:
do the teardown instructions get more simple now with these owner references and automatic deletions by k8s?

Reply:
yes, i'll update the docs. No longer do we need to remove the api and osd roles, rolebindings, and serviceaccounts.